WooCommerce/Classes/ViewRelated/Orders/Order Creation/CustomerSection/OrderCustomerSection.swift

```diff
@@ -66,8 +64,7 @@ private struct OrderCustomerSectionContent: View {
         }
         .padding(.horizontal, insets: safeAreaInsets)
         .background(Color(.listForeground))
-
-        Divider()
+        .addingTopAndBottomDividers()
```

Comment:
thinking out loud:
I wonder if something like `dividing(.top, .bottom)` makes sense 🤔

Reply:
Agree, I was thinking about passing edges here, but not sure if that's useful. Feel free to update modifier when you need more specialized version.